favorites.py

```diff
@@ -103,12 +103,24 @@ class FavFileMgr(object):
     """Handle file actions."""
 
     @classmethod
+    def fav_file(cls, filename, data=False, read=False, write=False):

```

Comment:
Moving these to their own function is fine, but I'm not big on this function behaving different based on read or write.  In one case, it returns the data, in the other case it doesn't.  There isn't anything gained by having one function.  One called `read_fav_file` and one `write_fav_file` is fine.

Reply:
Will split them